src/include/flockmtl/core/functions/aggregate/llm_rerank.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "flockmtl/common.hpp"
+#include <nlohmann/json.hpp>
+
+namespace flockmtl {
+namespace core {
+
+class LlmReranker {
+public:
+    string model;
```

Comment:
Many of these fields should also be private I think.

Reply:
DONE